react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -95,99 +100,55 @@ const CreateMapModal = ({ isOpen, toggle }: CreateMapModalProps) => {
-                {touched.system_file && errors.system_file && (
-                  <div className={`${styles['custom-error-message']}`}>
-                    {errors.system_file}
-                  </div>
-                )}
-              </FormGroup>
-              {/* TODO_REACT: This part will change once the FileBrowser component is added. https://tacc-main.atlassian.net/browse/WG-208*/}
-              <FormGroup className="row align-items-center">
-                <Label className="col-sm-4">Save Location:</Label>
-                <div className="col-sm-8 text-primary">
-                  {userData ? `/${userData.username}` : 'Loading...'}
+                <div className={`${styles['flexContainer-alt']}`}>
+                  <label htmlFor="save-location-label">Save Location:</label>
+                  <span className="text-primary">/{userData?.username}</span>
                 </div>
-              </FormGroup>
-              <FormGroup className="row mb-2">
-                <Label className="col-sm-4 col-form-label pt-0">
-                  Sync Folder:
-                </Label>
-                <div className="col-sm-8">
-                  <div className="form-check">
-                    <Field
-                      type="checkbox"
-                      name="syncFolder"
-                      id="syncFolder"
-                      className="form-check-input"
-                      checked={values.syncFolder}
-                      onChange={handleChange}
-                    />
-                  </div>
-                  <br />
-                  <br />
-                  <Label
-                    className={`${styles['form-check-label']}`}
-                    for="syncFolder"
-                  >
-                    When enabled, files in this folder are automatically synced
-                    into the map periodically.
-                  </Label>
+                <div className={`${styles['flexContainer']}`}>
+                  <label htmlFor="sync-folder-label">Sync Folder:</label>
+                  <span className={`${styles['check-wrapper']}`}>
+                    <FormikCheck name="syncFolder" label="" description="" />
+                  </span>
+                </div>
+                <div className={`${styles['custom-sync-description']}`}>
+                  When enabled, files in this folder are automatically synced
+                  into the map periodically.
                 </div>
-              </FormGroup>
+              </FieldWrapperFormik>
               {errorMessage && (
                 <div className={`${styles['custom-error-message']}`}>
                   {errorMessage}
                 </div>
               )}
               <ModalFooter className="justify-content-start">
-                <Button size="short" type="secondary" onClick={toggle}>
+                <Button size="short" type="secondary" onClick={handleToggle}>
```

Comment:
**Naming.**

Is the close button a toggle? I think not, because once you close it, you do not press the same button to "unclose" it. So… `handleClose`?